src/Components/Admins/Form/index.jsx

```diff
@@ -45,97 +120,100 @@ const Form = (props) => {
-            onChange={onChange}
-          />
+    <>
+      <form className={styles.myForm} onSubmit={onSubmit}>
+        <div className={styles.divContainer}>
+          <div className={styles.inputDiv}>
+            <label>First Name</label>
+            <Input name="firstName" type="text" value={formData.firstName} onChange={onChange} />
+          </div>
+          <div className={styles.inputDiv}>
+            <label>Last Name</label>
+            <Input
+              className={styles.input}
+              type="text"
+              name="lastName"
+              value={formData.lastName}
+              onChange={onChange}
+            />
+          </div>
+          <div className={styles.inputDiv}>
+            <label>DNI</label>
+            <Input
+              className={styles.input}
+              type="text"
+              name="dni"
+              value={formData.dni}
+              onChange={onChange}
+            />
+          </div>
+          <div className={styles.inputDiv}>
+            <label>Phone</label>
+            <Input
+              className={styles.input}
+              type="text"
+              name="phone"
+              value={formData.phone}
+              onChange={onChange}
+            />
+          </div>
+          <div className={styles.inputDiv}>
+            <label>Email</label>
+            <Input
+              className={styles.input}
+              type="text"
+              name="email"
+              value={formData.email}
+              onChange={onChange}
+            />
+          </div>
+          <div className={styles.inputDiv}>
+            <label>City</label>
+            <Input
+              className={styles.input}
+              type="text"
+              name="city"
+              value={formData.city}
+              onChange={onChange}
+            />
+          </div>
+          <div className={styles.inputDiv}>
+            <label>Password</label>
+            <div className={styles.password}>
+              <Input
+                className={styles.input}
+                type={showPassword ? 'text' : 'password'}
+                name="password"
+                value={formData.password}
+                onChange={onChange}
+              />
+              <FontAwesomeIcon
+                icon={showPassword ? faEyeSlash : faEye}
+                className={styles.showPasswordIcon}
+                onClick={togglePassword}
+              />
+            </div>
+          </div>
         </div>
-      </div>
-      <button className={styles.addButton} type="submit">
-        {switchButtonText}
-      </button>
-      <button className={styles.cancelButton} onClick={() => props.setShowForm(false)}>
-        Cancel
-      </button>
-    </form>
+        <Button className={styles.addButton} type="confirm">
+          {switchButtonText}
+        </Button>
+        <Button type="cancel" className={styles.cancelButton} onClick={() => history.goBack()}>
```

Comment:
i have a problem with the history.goBack, for some reason if you open the edit form and just close it will take you back to the list but if you wait a sec it will take you back once again the page you where before the admin list